core-plugins/src/main/java/co/cask/hydrator/plugin/batch/action/SSHAction.java

```diff
@@ -116,49 +124,52 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
    */
   public static class SSHActionConfig extends PluginConfig {
     @Description("Command to be executed on the remote host. Should include filepath of script and any arguments")
+    @Macro
     private String command;
 
     @Description("Host name of the remote machine where the command needs to be executed.")
+    @Macro
     private String host;
 
     @Nullable
     @Description("Port to connect to. Defaults to 22")
+    @Macro
     private Integer port;
 
     @Description("User name used to connect to host")
+    @Macro
     private String user;
 
-    @Description("File path to Private key")
-    private String privateKeyFile;
+    @Description("Key used to lookup private key file from secure storage")
+    @Macro
+    private String privateKeyFileLookupKey;
 
     @Nullable
-    @Description("Password associated with private key")
-    private String password;
+    @Description("Passphrase associated with private key")
+    @Macro
+    private String passphrase;
 
     @Nullable
     @Description("Variable used to hold desired key name to be associated with script output in the instance " +
```

Comment:
Unrelated to your PR, but there are a couple of problems with this description:
1. Its hard to understand because of the long sentence.
2. It uses terms like ``ActionContext``, which are very code specific. 

What does it do? Also, is it too internal to be exposed to users?